### ISSUE TITLE
[1.20.1]似乎在处理 DistanceManager 和 ServerChunkCache 时引发 ConcurrentModificationException 异常导致服务器崩溃

### ISSUE THREAD

**Issue by windy664** [IzzelAliz/Arclight#1056]
### 请确认您已完成以下几件事

- [X] 正在使用最新的 Arclight
- [X] 已为所有插件及模组安装依赖
- [X] 已为所有插件及模组更新至最新版
- [X] 不可在 Spigot 复现
- [X] 不可在 Forge 复现

### Arclight 版本

arclight-forge-1.20.1-1.0.0-SNAPSHOT

### Java 版本 | 操作系统

zulu-jdk 19

### 相关 Mod/插件 的名称及版本

```raw
Mod List: 
		[前置][神秘主义][聪明大脑库]SmartBrainLib-fabric-1.20-1.11.ja|SmartBrainLib                 |smartbrainlib                 |1.11                |DONE      |Manifest: NOSIGNATURE
		[前置][动态联合]CyclopsCore-1.20.1-1.18.7.jar           |Cyclops Core                  |cyclopscore                   |1.18.7              |DONE      |Manifest: NOSIGNATURE
		[天空村庄] SkyVillages-1.0.2-1.20.1-forge-release.jar |Sky Villages                  |skyvillages                   |1.0.2-1.20.1-forge  |DONE      |Manifest: NOSIGNATURE
		[通电动力]energizedpower-1.20.1-1.7.6.jar             |Energized Power               |energizedpower                |1.20.1-1.7.6        |DONE      |Manifest: NOSIGNATURE
		[前置][炼金化学]alchemylib-1.20.1-1.0.29.jar            |AlchemyLib                    |alchemylib                    |1.0.29              |DONE      |Manifest: NOSIGNATURE
		[前置][合成扩展]Cucumber-1.20.1-7.0.0.jar               |Cucumber Library              |cucumber                      |7.0.0               |DONE      |Manifest: NOSIGNATURE
		[前置]geckolib-forge-1.20.1-4.2.1.jar               |GeckoLib 4                    |geckolib                      |4.2.1               |DONE      |Manifest: NOSIGNATURE
		[更多灯笼] additionallanterns-1.0.4-forge-mc1.20.jar  |Additional Lanterns           |additionallanterns            |1.0.4               |DONE      |Manifest: NOSIGNATURE
		[前置][铁魔法]player-animation-lib-forge-1.0.2-rc1+1.20|Player Animator               |playeranimator                |1.0.2-rc1+1.20      |DONE      |Manifest: NOSIGNATURE
		[JEI物品管理器]jei-1.20.1-forge-15.2.0.23.jar          |Just Enough Items             |jei                           |15.2.0.23           |DONE      |Manifest: NOSIGNATURE
		[合成拓展] ExtendedCrafting-1.20.1-6.0.1.jar          |Extended Crafting             |extendedcrafting              |6.0.1               |DONE      |Manifest: NOSIGNATURE
		[更多实用工具]scalar_utils-1.20.1-5.4.10.jar            |Scalar Utils 5: Definitive Edi|scalar_utils                  |1.20.1-5.4.10       |DONE      |Manifest: NOSIGNATURE
		[网络发包修复]connectivity-1.20.1-4.4.jar               |Connectivity Mod              |connectivity                  |1.20.1-4.4          |DONE      |Manifest: NOSIGNATURE
		[是，史蒂夫模型] yesstevemodel-1.20-release-1.1.5-hotfix-|Yes Steve Model               |yes_steve_model               |1.1.5-hotfix-2      |DONE      |Manifest: NOSIGNATURE
		[前置][铁魔法]caelus-forge-3.1.0+1.20.jar              |Caelus API                    |caelus                        |3.1.0+1.20          |DONE      |Manifest: NOSIGNATURE
		[更多结构]Structory_1.20.1_v1.3.2.jar                 |Structory                     |structory                     |1.3.2               |DONE      |Manifest: NOSIGNATURE
		[懒人厨房] cookingforblockheads-forge-1.20-16.0.0.jar |CookingForBlockheads          |cookingforblockheads          |16.0.0              |DONE      |Manifest: NOSIGNATURE
		[摩登生活]modernlife-1.20.1-1.64.jar                  |Modern Life                   |modernlife                    |1.20.1-1.64         |DONE      |Manifest: NOSIGNATURE
		[自然罗盘／生物群系指南针] NaturesCompass-1.20.1-1.11.1-forge.|Nature's Compass              |naturescompass                |1.0.0               |DONE      |Manifest: NOSIGNATURE
		[前置][柠檬核心]lemon_core-1.20.1-1.0.21.jar            |Lemon Core                    |lemon_core                    |1.20.1-1.0.21       |DONE      |Manifest: NOSIGNATURE
		[更多光源] additional_lights-1.20.1-2.1.7.jar         |Additional Lights             |additional_lights             |2.1.7               |DONE      |Manifest: NOSIGNATURE
		[魔法使]mahoutsukai-1.20.1-v1.34.53.jar              |Mahou Tsukai                  |mahoutsukai                   |1.20.1-v1.34.53     |DONE      |Manifest: NOSIGNATURE
		[前置][懒人厨房]balm-forge-1.20-7.0.4.jar               |Balm                          |balm                          |7.0.4               |DONE      |Manifest: NOSIGNATURE
		[合成表]CraftTweaker-forge-1.20.1-14.0.10.jar        |CraftTweaker                  |crafttweaker                  |14.0.10             |DONE      |Manifest: NOSIGNATURE
		[更多磁盘] ExtraDisks-1.20.1-3.0.0.jar                |Extra Disks                   |extradisks                    |1.20.1-3.0.0        |DONE      |Manifest: NOSIGNATURE
		[原神仪器]genshinstrument-1.20.1-3.1.jar              |Genshin Instruments           |genshinstrument               |3.1                 |DONE      |Manifest: NOSIGNATURE
		[前置]cloth-config-11.0.99-forge.jar                |Cloth Config v10 API          |cloth_config                  |11.0.99             |DONE      |Manifest: NOSIGNATURE
		forge-1.20.1-47.1.1-universal.jar                 |Forge                         |forge                         |47.1.1              |DONE      |Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		[精致存储] refinedstorage-1.12.2.jar                  |Refined Storage               |refinedstorage                |1.12.2              |DONE      |Manifest: NOSIGNATURE
		[更多箱子]ironchest-1.20.1-14.4.4.jar                 |Iron Chests                   |ironchest                     |1.20.1-14.4.4       |DONE      |Manifest: NOSIGNATURE
		[集成合成学] IntegratedCrafting-1.20.1-1.1.3.jar       |IntegratedCrafting            |integratedcrafting            |1.1.3               |DONE      |Manifest: NOSIGNATURE
		[炼金化学]alchemistry-1.20.1-2.3.3.jar                |Alchemistry                   |alchemistry                   |2.3.3               |DONE      |Manifest: NOSIGNATURE
		server-1.20.1-20230612.114412-srg.jar             |Minecraft                     |minecraft                     |1.20.1              |DONE      |Manifest: NOSIGNATURE
		[农夫乐事] FarmersDelight-1.20.1-1.2.2a.jar           |Farmer's Delight              |farmersdelight                |1.20.1-1.2.2a       |DONE      |Manifest: NOSIGNATURE
		[前置][集成动力]CommonCapabilities-1.20.1-2.9.0.jar     |CommonCapabilities            |commoncapabilities            |2.9.0               |DONE      |Manifest: NOSIGNATURE
		[前置][生物群系]TerraBlender-forge-1.20.1-3.0.0.169.jar |TerraBlender                  |terrablender                  |3.0.0.169           |DONE      |Manifest: NOSIGNATURE
		[汉堡包]burgermod-2.9.4-1.20.1.jar                   |Burger Mod                    |burgermod                     |2.9.4               |DONE      |Manifest: NOSIGNATURE
		[前置][妖怪之山通用库] mysterious_mountain_lib-1.2.7-1.20.j|Mysterious Mountain Lib       |mysterious_mountain_lib       |1.2.7-1.20          |DONE      |Manifest: NOSIGNATURE
		[更多生物群系]BiomesOPlenty-1.20.1-18.0.0.592.jar       |Biomes O' Plenty              |biomesoplenty                 |18.0.0.592          |DONE      |Manifest: NOSIGNATURE
		[前置][创造核心]CreativeCore_FORGE_v2.10.31_mc1.20.1.jar|CreativeCore                  |creativecore                  |2.10.31             |DONE      |Manifest: NOSIGNATURE
		[前置]supermartijn642corelib-1.1.9-forge-mc1.20.jar |SuperMartijn642's Core Lib    |supermartijn642corelib        |1.1.9               |DONE      |Manifest: NOSIGNATURE
		[炼金术]theurgy-1.20.1-1.3.10.jar                    |Theurgy                       |theurgy                       |1.3.10              |DONE      |Manifest: NOSIGNATURE
		[植物魔法]Botania-1.20.1-441-FORGE-SNAPSHOT.jar       |Botania                       |botania                       |1.20.1-441-FORGE-SNA|DONE      |Manifest: NOSIGNATURE
		[倒置火把] ceilingtorch-1.20-1.25.jar                 |Ceiling Torch                 |ceilingtorch                  |1.25                |DONE      |Manifest: NOSIGNATURE
		[末影接口]EnderIO-6.1.1453-alpha-nightly-all.jar      |Ender IO                      |enderio                       |6.1.1453-alpha-night|DONE      |Manifest: NOSIGNATURE
		[前置][饰品栏]curios-forge-5.2.0-beta.3+1.20.1.jar     |Curios API                    |curios                        |5.2.0-beta.3+1.20.1 |DONE      |Manifest: NOSIGNATURE
		[通用机械]Mekanism-1.20.1-10.3.9.homebaked-all.jar    |Mekanism                      |mekanism                      |10.3.9              |DONE      |Manifest: NOSIGNATURE
		[赛🐎娘]umapyoi-0.2.15-1.20.jar                     |Umapyoi                       |umapyoi                       |0.2.15-1.20         |DONE      |Manifest: NOSIGNATURE
		[管道]pipez-1.20.1-1.1.3.jar                        |Pipez                         |pipez                         |1.20.1-1.1.3        |DONE      |Manifest: NOSIGNATURE
		flywheel-forge-1.20-0.6.9-4.jar                   |Flywheel                      |flywheel                      |0.6.9-4             |DONE      |Manifest: NOSIGNATURE
		[机械动力] create-1.20.1-0.5.1.d.jar                  |Create                        |create                        |0.5.1.d             |DONE      |Manifest: NOSIGNATURE
		[前置][帕特抽手册]Patchouli-1.20.1-82-FORGE-SNAPSHOT.jar |Patchouli                     |patchouli                     |1.20.1-82-FORGE-SNAP|DONE      |Manifest: NOSIGNATURE
		[动态联合]IntegratedDynamics-1.20.1-1.16.9.jar        |IntegratedDynamics            |integrateddynamics            |1.16.9              |DONE      |Manifest: NOSIGNATURE
		[前置][化学]chemlib-1.20.1-2.0.17.jar                 |ChemLib                       |chemlib                       |2.0.17              |DONE      |Manifest: NOSIGNATURE
		[作物胜景]Croptopia-1.20.1-FORGE-2.3.2.jar            |Croptopia                     |croptopia                     |2.3.2               |DONE      |Manifest: NOSIGNATURE
		[照相机]camera-1.20.1-1.0.4.jar                      |Camera Mod                    |camera                        |1.20.1-1.0.4        |DONE      |Manifest: NOSIGNATURE
		[法术书]irons_spellbooks-1.20.1-1.1.8.jar            |Iron's Spells 'n Spellbooks   |irons_spellbooks              |1.20.1-1.1.8        |DONE      |Manifest: NOSIGNATURE
		[前置][手册]modonomicon-1.20.1-forge-1.38.5.jar       |Modonomicon                   |modonomicon                   |1.38.5              |DONE      |Manifest: NOSIGNATURE
		[飞车奇匠] automobility-0.4.2+1.20.1-forge.jar        |Automobility                  |automobility                  |0.4.2+1.20.1-forge  |DONE      |Manifest: NOSIGNATURE
		[前置]architectury-9.0.8-forge.jar                  |Architectury                  |architectury                  |9.0.8               |DONE      |Manifest: NOSIGNATURE
		[铁氧体磁芯] ferritecore-6.0.0-forge.jar               |Ferrite Core                  |ferritecore                   |6.0.0               |DONE      |Manifest: 41:ce:50:66:d1:a0:05:ce:a1:0e:02:85:9b:46:64:e0:bf:2e:cf:60:30:9a:fe:0c:27:e0:63:66:9a:84:ce:8a
		[神秘主义]occultism-1.20.1-1.81.6.jar                 |Occultism                     |occultism                     |1.81.6              |DONE      |Manifest: NOSIGNATURE
		[卡哇伊]KawaiiDishes_1.10.1-1.20.1.jar               |Kawaii Dishes                 |kawaiidishes                  |1.10.1              |DONE      |Manifest: NOSIGNATURE
		[浩渺能源]VoluminousEnergy-1.20.1-0.4.0.0.jar         |Voluminous Energy             |voluminousenergy              |1.20.1-0.4.0.0      |DONE      |Manifest: NOSIGNATURE
		arclight-1.20.1-1.0.0-SNAPSHOT-0676095.jar        |Arclight Mod                  |arclight                      |1.20.1-1.0.0-SNAPSHO|DONE      |Manifest: NOSIGNATURE
		[前置][循环]flib-1.20.1-0.0.8.jar                     |flib                          |flib                          |0.0.8               |DONE      |Manifest: 1f:47:ac:b1:61:82:96:b8:47:19:16:d2:61:81:11:60:3a:06:4b:61:31:56:7d:44:31:1e:0c:6f:22:5b:4c:ed
		[精致存储附属] refinedstorageaddons-0.10.0.jar          |Refined Storage Addons        |refinedstorageaddons          |0.10.0              |DONE      |Manifest: NOSIGNATURE
		[跑图延迟修复]chunksending-1.20.1-2.5.jar               |chunksending mod              |chunksending                  |1.20.1-2.5          |DONE      |Manifest: NOSIGNATURE
		[豆腐工艺]TofuCraftReload-1.20.1-5.2.6.0.jar          |TofuCraftReload               |tofucraft                     |1.20.1-5.2.6.0      |DONE      |Manifest: NOSIGNATURE
		[循环]Cyclic-1.20.1-1.12.1.jar                      |Cyclic                        |cyclic                        |1.12.1              |DONE      |Manifest: 
plugins:
Plugins (56): Law, PlaceholderAPI, CustomPlaceholder, Story, BBSToper, Ip2region, NBTAPI, MiniMOTD, LuckPerms, CrazyAuctionsPlus, PlayerChat, CMILib, ItemNBTEdit, spark, LoneLibs, AlwaysKeepInventory, ProtocolLib, Uniporter, LiteSignIn, MiaoBoard, Vault, WTF, SbPlaceholder, BKCommonLib, PlayerPoints, ItemCommand, SkinsRestorer, PlayerTitle, DemonMarket, MyCommand, WorldEdit, PlayerKits, DecentHolograms, Multiverse-Core, Citizens, AuthMe, SakuraPurchasePlugin, CMI, AllMusic, CMIEInjector, PlayerGuild, CoreProtect, PlayerTop, DreamCleaner, Towny, PocketGames, Residence, BlockLocker, ItemsAdder, BanItem, TownyGUI, TrMenu, YeqiLib, DeluxeMenus, NekoMaid, PlugManX (PlugMan)
```

### 错误描述

在尝试启动服务器时，遇到了一次崩溃。崩溃报告中显示，问题可能与 DistanceManager 和 ServerChunkCache 类有关。怀疑是 Arclight 引起了这次崩溃。

### 复现步骤

1.安装最新版本的 Arclight 
2.启动服务器
3.过一段时间后，服务器在启动过程中崩溃

### 报错信息

[crash-2023-07-30_12.38.35-server.txt](https://github.com/IzzelAliz/Arclight/files/12208543/crash-2023-07-30_12.38.35-server.txt)

**Comment by Shybella** [comment]
This is also occurring on Direwolf20 1.19.2.

**Comment by windy664** [comment]
> This is also occurring on Direwolf20 1.19.2.

所以说，是模组的问题吗？

**Comment by Shybella** [comment]
> > This is also occurring on Direwolf20 1.19.2.
> 
> 所以说，是模组的问题吗？

Yes but I think it the past it was solved within the hybrid software

**Comment by windy664** [comment]
> > > This is also occurring on Direwolf20 1.19.2.
> > 
> > 
> > 所以说，是模组的问题吗？
> 
> Yes but I think it the past it was solved within the hybrid software

看来，那只能等arclight修复了 [脑瓜疼]

**Comment by IzzelAliz** [comment]
如果你能给我一个稳定复现的步骤，我就可以修

**Comment by windy664** [comment]
> 如果你能给我一个稳定复现的步骤，我就可以修

你好，我刚刚又出现了这个问题
[crash-2023-08-02_21.51.10-server.txt](https://github.com/IzzelAliz/Arclight/files/12240865/crash-2023-08-02_21.51.10-server.txt)

复现步骤：
1.服务器TPS过低，经跑spark取证https://spark.lucko.me/gMnFWt3dhy 基本都是循环物品卡服
2.修改循环mod的config，将他们的机器花费电量跳到一个最大值，且相应时间也最久，相当于间接等效禁用了这类机器
3.重启服务器，加载有关机器的区块
4.崩服

以下是我的循环mod的配置文件

[cyclic.txt](https://github.com/IzzelAliz/Arclight/files/12240890/cyclic.txt)
虽然说这个问题之前也出现过 但只是偶尔出现 大概4~6天出现一次

我猜你可能需要使用特殊的数据结构

**Comment by IzzelAliz** [comment]
那么稳定吗，你启动就能崩？

**Comment by Shybella** [comment]
> Is it so stable that you can crash when you start it?

It's hard to replicate but happens often, I think it's because multiple blocks accessing the same block holding some item at the same time. 

I think chunk loaders are also increasing the frequency of it too somehow.

**Comment by windy664** [comment]
> 那么稳定吗，你启动就能崩？

不一定 只是偶尔会出现 就几天一次 很头疼

**Comment by Shybella** [comment]
https://github.com/MinecraftForge/MinecraftForge/pull/5861 

Might be related, this was fixed in Spongeforge at some point. I know this solved it for me back then

**Comment by windy664** [comment]
头疼呐 服务器重启进服又复现一次
[crash-2023-08-03_07.22.27-server.txt](https://github.com/IzzelAliz/Arclight/files/12245445/crash-2023-08-03_07.22.27-server.txt)

**Comment by windy664** [comment]
[server.properties.txt](https://github.com/IzzelAliz/Arclight/files/12245458/server.properties.txt)
顺便补充一下

**Comment by windy664** [comment]
> 那么稳定吗，你启动就能崩？

海螺啊，我又出現這個問題了，真係好煩啊

這個係我日志
`[098月2023 11:15:43.784] [Server thread/INFO] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: HHHHastur issued server command: /menu
[098月2023 11:15:44.541] [Server thread/ERROR] [net.minecraft.world.entity.ai.Brain/]: No key memories in MapLike[{}]
[098月2023 11:15:44.541] [Server thread/ERROR] [net.minecraft.world.entity.ai.Brain/]: No key memories in MapLike[{}]
[098月2023 11:15:44.543] [Server thread/ERROR] [net.minecraft.server.MinecraftServer/]: Encountered an unexpected exception
java.util.ConcurrentModificationException: null
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1605) ~[?:?] {}
	at java.util.HashMap$KeyIterator.next(HashMap.java:1628) ~[?:?] {}
	at java.lang.Iterable.forEach(Iterable.java:74) ~[?:?] {re:mixin,re:mixin,re:mixin,re:mixin,re:mixin}
	at net.minecraft.server.level.DistanceManager.m_140805_(DistanceManagerMixin.java:119) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.arclight.core.json:server.level.DistanceManagerMixin,pl:mixin:A,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.level.ServerChunkCache.m_8489_(ServerChunkCacheMixin.java:273) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.arclight.core.json:server.level.ServerChunkCacheMixin,pl:mixin:A,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.level.ServerChunkCache.bridge$tickDistanceManager(ServerChunkCacheMixin.java) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.arclight.core.json:server.level.ServerChunkCacheMixin,pl:mixin:A,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.level.ServerChunkCache$MainThreadExecutor.m_7245_(ServerChunkCache_MainThreadExecutorMixin.java:631) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?] {re:mixin,re:classloading,pl:mixin:APP:mixins.arclight.core.json:server.level.ServerChunkCache_MainThreadExecutorMixin,pl:mixin:A,re:mixin}
	at net.minecraft.server.level.ServerChunkCache.m_8466_(ServerChunkCacheMixin.java:269) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.arclight.core.json:server.level.ServerChunkCacheMixin,pl:mixin:A,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_129961_(MinecraftServerMixin.java:758) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.arclight.core.json:server.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:A,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_7245_(MinecraftServerMixin.java:747) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.arclight.core.json:server.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:A,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.util.thread.BlockableEventLoop.m_18699_(BlockableEventLoop.java:115) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_130012_(MinecraftServerMixin.java:732) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.arclight.core.json:server.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:A,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServerMixin.java:2074) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.arclight.core.json:server.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:A,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServerMixin.java:251) ~[server-1.20.1-20230612.114412-srg.jar%23303!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:mixins.arclight.core.json:server.MinecraftServerMixin,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer,pl:mixin:A,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at java.lang.Thread.run(Thread.java:1589) ~[?:?] {re:mixin,re:mixin,re:mixin}
[098月2023 11:15:49.711] [Server thread/FATAL] [net.minecraftforge.common.ForgeMod/]: Preparing crash report with UUID c092b96d-6ba3-4be8-9f7a-5e487528e28d`

完整版在

[2023-08-09-1.txt](https://github.com/IzzelAliz/Arclight/files/12298012/2023-08-09-1.txt)

錯誤報告：
[crash-2023-08-09_11.15.49-server.txt](https://github.com/IzzelAliz/Arclight/files/12297956/crash-2023-08-09_11.15.49-server.txt)

**Comment by Shybella** [comment]
Can this be back ported to 1.19.2? Thanks

**Comment by windy664** [comment]
谢谢您的修复，请问在何处下载？

**Comment by vgrynch** [comment]
I also would like this fixed for 1.19.2, happens for too often :(

**Comment by vgrynch** [comment]
This also probably related to https://github.com/IzzelAliz/Arclight/issues/993
